src/dash/student_view/TutorSearchResult.jsx

```diff
@@ -1,7 +1,5 @@
 import React from 'react';
 
-
-
 class TutorSearchResult extends React.Component {
     constructor(props) {
         super(props);
```

Comment:
in your render function you are directly mutating the state. That's bad. You should be using `this.setState({})`